importer/helper.go

```diff
@@ -92,7 +81,23 @@ func (h *ImportHelper) Import(db idb.IndexerDb, args []string) {
 		h.Log.Infof("%d blocks in %s, %.0f/s, %d txn, %.0f/s", blocks, dt.String(), float64(time.Second)*float64(blocks)/float64(dt), txCount, float64(time.Second)*float64(txCount)/float64(dt))
 	}
 
-	accountingRounds, txnCount := updateAccounting(db, h.GenesisJSONPath, h.NumRoundsLimit, h.Log)
+	initialImport := InitialImport(db, h.GenesisJSONPath, h.Log)
+	maybeFail(err, h.Log, "problem getting the max round")
+	filter := idb.UpdateFilter{
+		StartRound: -1,
+	}
+	if h.NumRoundsLimit != 0 {
+		filter.RoundLimit = &h.NumRoundsLimit
+	}
+	if !initialImport {
+		state, err := db.GetImportState()
+		maybeFail(err, h.Log, "problem getting the import state")
+		filter.StartRound = state.AccountRound
+	}
+	accountingRounds, txnCount := updateAccounting(db, filter, h.Log)
+	if initialImport {
+		accountingRounds++
+	}
```

Comment:
Since we're deprecating `import` I'm not too concerned about how ugly this became.